scripts/deploy.sh

```diff
@@ -28,7 +28,7 @@ REPO_DIR="$(git rev-parse --show-toplevel)"
 
 cd "$REPO_DIR"
 
-SOURCE_FILE="source.tar.bz2"
+SOURCE_FILE="source.tar.gz"
```

Comment:
Why change it to `.gz`?

I tried looking up what Nix expects, and nix-channel states it expects `nixexprs.tar.xz`, confusing me even further.

Reply:
Because thats what git archive supports out-of-the-box and I didn't care that much whether to use gzip or bzip2. Nix can handle all kind of compressions.